cgroups/systemd/apply_systemd.go

```diff
@@ -350,6 +350,20 @@ func (m *Manager) Set(container *configs.Config) error {
 	panic("not implemented")
 }
 
+func (m *Manager) EnterProcess(pid int) error {
```

Comment:
Unfortunately it's not easy. We can make a separate function and have it depend on that, but I'm not sure its worth it for a short snippet.